t/17-labels_carry_over.t

```diff
@@ -0,0 +1,110 @@
+# Copyright (C) 2016 SUSE Linux GmbH
```

Comment:
SUSE Linux GmbH and SUSE LLC are both valid

SUSE Linux Products GmbH is no longer valid
On 24 Feb 2016 7:18 pm, "Oliver Kurz" <notifications@github.com> wrote:

> In t/17-labels_carry_over.t
> <https://github.com/os-autoinst/openQA/pull/564#discussion_r53981489>:
>
> > @@ -0,0 +1,110 @@
> > +# Copyright (C) 2016 SUSE Linux GmbH
>
> are you sure:
>
> $ git grep -h Copyright | sort | uniq --count | sort -n | tail -n 10
>       2 # Copyright (C) 2016 Red Hat
>       2 # Copyright (C) 2016 SUSE Linux GmbH
>       3 # Copyright (C) 2015 Red Hat
>       3 # Copyright (C) 2016 SUSE LLC
>       6 # Copyright (c) 2015 SUSE LINUX GmbH, Nuernberg, Germany.
>       8  * Copyright jQuery Foundation and other contributors
>      10 # Copyright (C) 2015 SUSE LLC
>      11 # Copyright (C) 2015 SUSE Linux Products GmbH
>      35 # Copyright (C) 2015 SUSE Linux GmbH
>      92 # Copyright (C) 2014 SUSE Linux Products GmbH
>
> ;-P
>
> I would not mind a copyright checker script, until then …
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/os-autoinst/openQA/pull/564/files#r53981489>.
>